spring-ai-client-chat/src/main/java/org/springframework/ai/chat/client/DefaultChatClient.java

```diff
@@ -659,18 +663,36 @@ public static class DefaultChatClientRequestSpec implements ChatClientRequestSpe
 			this(ccr.chatModel, ccr.userText, ccr.userParams, ccr.userMetadata, ccr.systemText, ccr.systemParams,
 					ccr.systemMetadata, ccr.toolCallbacks, ccr.toolCallbackProviders, ccr.messages, ccr.toolNames,
 					ccr.media, ccr.chatOptions, ccr.advisors, ccr.advisorParams, ccr.observationRegistry,
-					ccr.observationConvention, ccr.toolContext, ccr.templateRenderer);
+					ccr.chatClientObservationConvention, ccr.toolContext, ccr.templateRenderer,
+					ccr.advisorObservationConvention);
 		}
 
+		/**
+		 * @deprecated in favor of the other constructor.
+		 */
+		@Deprecated(since = "1.1.0", forRemoval = true)
 		public DefaultChatClientRequestSpec(ChatModel chatModel, @Nullable String userText,
 				Map<String, Object> userParams, Map<String, Object> userMetadata, @Nullable String systemText,
 				Map<String, Object> systemParams, Map<String, Object> systemMetadata, List<ToolCallback> toolCallbacks,
 				List<ToolCallbackProvider> toolCallbackProviders, List<Message> messages, List<String> toolNames,
```

Comment:
Slightly related to this PR: during rebasing Thomas' changes, I found an extra param here: `List<ToolCallbackProvider> toolCallbackProviders` that wasn't there before, I guess that's  a breaking change between 1.0 and 1.1. If that's ok, maybe the deprecated ctor can be removed too in this PR? (I prefer not breaking things though, that's why I kept it.)